.gitattributes

```diff
@@ -1,3 +1,6 @@
 * text eol=lf
 *.png binary
 *.jar binary
+
+# for newline tests this file needs crlf
+testlib/src/main/resources/lineEndings/JavaCode-WINDOWS.test eol=crlf
```

Comment:
Remove this, and all of the test resources.  It's better to create newline tests with explicit string constants, e.g.:

```java
String noTrailingNewline = "public class Java {}"
String hasTrailingNewline = noEndingNewline + "\n";
setFile("path").toContent(noTrailingNewline);
runTest();
assertFile("path").hasContent(hasTrailingNewline);
```

Most editors (github included) makes it hard to see trailing newlines and unix/windows differences.  Best to just write them explicitly when you're writing a test for them.

Reply:
I totaly agree.

Reply:
fixed for newline tests